data/hai/hai news.txt

```diff
@@ -165,3 +165,131 @@ news "reformed pirate in Hai space"
 			"The Hai will pay you to rush cargo to human space and back. Don't die trying."
 		word
 			`"`
+
+news "unfettered hai dialog"
+	location
+		government "Hai (Unfettered)"
+	name
+		word
+			"Unfettered fighter"
+			"Unfettered youth"
+			"Unfettered warlord"
+			"Hai youth"
+			"Hai warlord"
+	message
+		word
+			`"`
+		word
+			"How did you get here? You should've been blasted to bits by our ships."
+			"I came here because this is the only place in the galaxy where I feel wanted."
+			"Weak human. Why do you dare to come to the land of the true Hai?"
+			"Altered Hai are soft and weak-willed. They care not for true struggle or challenge."
+			"The last time a human called me 'cute', I ran him through with a machete."
+			"I seek challenge, and what greater challenge is there than to oppose the Drak, those with the power of gods?"
+			"Sometimes I miss my family and friends back home, but then I remember that here I am a respected warrior. Back there I was a slave to the Drak."
+			"Our ships can beat anything you stupid monkeys can make."
+			"This sure beats being a factory worker."
+			"Without change, a civilization will decay and fall. This will happen to the fettered Hai, as it has happened to countless empires throughout history."
+			"What are you looking at, monkey?"
+			"I can take on a human like you any day."
+			"You looking for a fight, monkey?"
+		word
+			`"`
+	name
+		word
+			"Unfettered fighter"
+			"Unfettered youth"
+			"Hai youth"
+	message
+		word
+			`"`
+		word
+			"At first I thought that joining the Unfettered would be fun. Now I can't go back or I'll be killed."
```

Comment:
The various other texts which talk about how the Hai view the Unfettered suggest that this is in face entirely false.

Reply:
I meant to imply that other Unfettered would try to kill him for 'deserting'.

Reply:
Ah yeah that was not at all clear. I'm really not inclined to think that *death* is the likely outcome though, even on the Unfettered side of things.

Reply:
I'll try fixing this tomorrow then.

Reply:
I edited it, is it okay now?